wagtailio/project_styleguide/templates/patterns/components/streamfields/headline/headline.yaml

```diff
@@ -4,7 +4,7 @@ context:
     subheading: Created by the digital agency Torchbox in 2014
     intro: Once Wagtail was released as an open source project, 46 external contributors participated in creating version 1.0. Since then, hundreds of contributors from around the world have contributed to Wagtail. The first Wagtail conference, called Wagtail Space, was held in March 2017 in Arnhem in the Netherlands. The name Wagtail comes from the pretty little birds that gather on the lawns outside Torchbox’s offices from spring to autumn. Torchbox wanted a charming name that had a sense of light and delight to it. Plus, birds make easy logos.
     cta:
-      text: Find out more
+      cta_text: Find out more
```

Comment:
Note: It's not a major issue but the naming convention of this is a little redundant, the prefix is the model name which we already know as it's the block the editor is using.

Reply:
Makes sense! Done in ae8d891 and 376e73b